apps/example-orchestrator-ui/components/Subscription/subscriptionQuery.ts

```diff
@@ -210,9 +145,12 @@ export function mapApiResponseToSubscriptionDetail(
         ),
         note: subscription?.note ?? '',
         product: product,
-        fixedInputs: subscription.fixedInputs,
-        customer: customer,
```

Comment:
No customer in a subscription?

I though the backend would handle at least a name + customer ID
(fixed/hardcoded when not needed)

Reply:
Currently the backend does not seem to support the customer prop

Reply:
Added reference to the story as a todo that will add the prop back